codegenerator/cli/templates/static/codegen/src/db/Migrations.res

```diff
@@ -11,7 +11,10 @@ let creatTableIfNotExists = (sql, table) => {
       let fieldName = field->Table.getDbFieldName
 
       {
-        `"${fieldName}" ${(fieldType :> string)}${isArray ? "[]" : ""}${switch defaultValue {
+        `"${fieldName}" ${switch fieldType {
+          | Custom(name) if !(name->Js.String2.startsWith("NUMERIC(")) => `"${Env.Db.publicSchema}".${name}`
```

Comment:
This seems a bit fragile and I'm not following. When does custom start with "NUMERIC(" ?

Reply:
I kind of think we should rather separate these into two variant constructors. Maybe Enum and Custom for eg.

Reply:
Ok cool 👍🏼 maybe just add a comment explaining then.